modules/getting-started/p_register-client.adoc

```diff
@@ -0,0 +1,33 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to register my first client
+
+[id="register-client"]
+= Registering the {appserver_name} application
+
+After you have deployed the sample code, you can define and register the client in the {project_name} Admin Console.
```

Comment:
says 'deployed' here but uses 'installed' other places in the doc - maybe keep this consistent, i.e. is the sample code deployed or installed?

Reply:
Yes, good point. I standardized on install.